bootstrap/linux-9.0.2.json

```diff
@@ -88,15 +88,27 @@
     "dependencies": [
         {
             "cabal_sha256": "1125a0a4be3aafc8da208940f219d4e4df8a0db87d892cc42bb369071855c590",
+            "component": "lib:directory",
             "flags": [],
             "package": "directory",
             "revision": 0,
             "source": "hackage",
             "src_sha256": "dc2785d6548cec2e80700fab007d3e9467f65d3c58ab3efa21b34d9017cf0efd",
             "version": "1.3.7.1"
         },
+        {
+            "cabal_sha256": "0ab9095e18bcace5adf1d07fcaa489ae4d5c141e9c0fcfeb5343362d04b9dc5b",
+            "component": "exe:alex",
+            "flags": [],
+            "package": "alex",
+            "revision": 0,
+            "source": "hackage",
+            "src_sha256": "810f8e85ea6b87c37cba10f7660d7f1aa0ba251c1275e3a18c312964bb329a63",
+            "version": "3.3.0.0"
```

Comment:
Ditto why not 3.4.0.0?

Reply:
See [above](https://github.com/haskell/cabal/pull/8980#discussion_r1261903307).